celery/app/task.py

```diff
@@ -1113,8 +1113,8 @@ def add_trail(self, result):
             self.request.children.append(result)
         return result
 
-    def push_request(self, *args, **kwargs):
-        self.request_stack.push(Context(*args, **kwargs))
+    def push_request(self, **kwargs):
```

Comment:
should we keep the *args?

Reply:
I restored the args. 
It is a bit misleading since args can only be a single dictionary otherwise it fails. 
I did not find any occurence using it like this, but I leave it in any case.